### PR TITLE
Fix: size variable in ndctl test

### PR DESCRIPTION
**Pull request by harish-24**
Patch fixes size variable in ndctl test

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by harish-24** [comment]
```
 (07/10) ndctl.py:NdctlTest.test_multiple_ns_modes_region:  PASS (15.43 s)
```

File: memory/ndctl.py [hunk 1]
```diff
@@ -358,9 +358,9 @@ def test_multiple_ns_multiple_region(self):
             self.log.info("Using %s for muliple namespaces", region)
             self.log.info("Creating %s namespaces", self.cnt)
             if not self.size:
-                size = self.get_json_val(self.get_json(
+                self.size = self.get_json_val(self.get_json(
```
Comment (PraveenPenguin) [inline memory/ndctl.py]: one question why we need to change scope of variable as seems me it used only one place or it is bug fix ?

Comment (harish-24) [reply memory/ndctl.py]: yes, it is a fix.

Comment (PraveenPenguin) [reply memory/ndctl.py]: ok sure

**Pull request merged**